src/main/webapp/catSortable.js

```diff
@@ -54,7 +54,7 @@ var CategorizedSortable = (function () {
         if (!firstRow) return;
 
         // We have a first row: assume it's the header, and make its contents clickable links
-        firstRow.each(function (cell) {
+        firstRow.forEach(function (cell) {
```

Comment:
All the changes in this file are loose copypasta from `war/src/main/webapp/scripts/sortable.js` in core.